src/shared/lib/comparison/ComparisonStore.ts

```diff
@@ -1035,6 +1064,34 @@ export default class ComparisonStore {
         []
     );
 
+    readonly survivalClinicalAttributesPrefix = remoteData({
+        await: () => [this.activeStudiesClinicalAttributes],
+        invoke: () => {
+            const attributes = getSurvivalAttributes(
+                this.activeStudiesClinicalAttributes.result!
+            );
+            let attributesPrefix: string[] = [];
+            // get paired attributes
+            attributes.forEach(attribute => {
```

Comment:
this is actually a reduce or filter operation since you are selectively pushing into attributesPrefix.    code would be clearer if you did ```attributesPrefix = arr.reduce or arr.filter```